deploy/windows/win.yaml

```diff
@@ -0,0 +1,130 @@
+        - args:
+            - '--csi-address=$(CSI_ENDPOINT)'
+            - '--probe-timeout=3s'
+            - '--health-port=29643'
+            - '--v=2'
+          command:
+            - /livenessprobe.exe
+          env:
+            - name: CSI_ENDPOINT
+              value: 'unix://C:\\csi\\csi.sock'
+          image: 'registry.k8s.io/sig-storage/livenessprobe:v2.7.0'
+          imagePullPolicy: IfNotPresent
+          name: liveness-probe
+          resources:
+            limits:
+              memory: 100Mi
+            requests:
+              cpu: 10m
+              memory: 40Mi
+          terminationMessagePath: /dev/termination-log
+          terminationMessagePolicy: File
+          volumeMounts:
+            - mountPath: 'C:\csi'
+              name: plugin-dir
+        - args:
+            - '--v=2'
+            - '--csi-address=$(CSI_ENDPOINT)'
+            - '--kubelet-registration-path=$(DRIVER_REG_SOCK_PATH)'
+          command:
+            - /csi-node-driver-registrar.exe
+          env:
+            - name: CSI_ENDPOINT
+              value: 'unix://C:\\csi\\csi.sock'
+            - name: DRIVER_REG_SOCK_PATH
+              value: 'C:\\var\\lib\\kubelet\\plugins\\pxd.portworx.com\\csi.sock'
+            - name: KUBE_NODE_NAME
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: spec.nodeName
+          image: 'registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.5.1'
+          imagePullPolicy: IfNotPresent
+          livenessProbe:
+            exec:
+              command:
+                - /csi-node-driver-registrar.exe
+                - '--kubelet-registration-path=$(DRIVER_REG_SOCK_PATH)'
+                - '--mode=kubelet-registration-probe'
+            failureThreshold: 3
+            initialDelaySeconds: 60
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 30
+          name: node-driver-registrar
+          resources:
+            limits:
+              memory: 100Mi
+            requests:
+              cpu: 10m
+              memory: 40Mi
+          terminationMessagePath: /dev/termination-log
+          terminationMessagePolicy: File
+          volumeMounts:
+            - mountPath: 'C:\var\lib\kubelet'
+              name: kubelet-dir
+            - mountPath: 'C:\csi'
+              name: plugin-dir
+            - mountPath: 'C:\registration'
+              name: registration-dir
+      dnsPolicy: ClusterFirst
+      hostNetwork: true
+      nodeSelector:
+        kubernetes.io/os: windows
+      priorityClassName: system-node-critical
+      restartPolicy: Always
+      schedulerName: default-scheduler
+      securityContext: {}
+      terminationGracePeriodSeconds: 30
+      tolerations:
+      - key: os
+        value: Windows
+      volumes:
+        - hostPath:
+            path: \\.\pipe\csi-proxy-filesystem-v1
+            type: ''
+          name: csi-proxy-fs-pipe-v1
```

Comment:
Looks like you copied this code from the SMB CSI Driver:
https://github.com/kubernetes-csi/csi-driver-smb/blob/d18eaf64b2ebd2ec299d1090c4262d02cd0ac161/deploy/csi-smb-node-windows.yaml#L120

Let's remove deadcode we're not using when copying external code

Reply:
Same for other volumes below... remove volumes that are not mounted anywhere